File: output/CodeGen/codenet/Python/Java/s554858080.java
public class s554858080 {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        String num = in.nextLine();
        if (num.length() % 4 == 0) {
            System.out.println("YES");
        } else {
            System.out.println("NO");
        }
    }
}

File: output/ChatGPT/codenet/Python/C++/s554858080.cpp
#include <iostream>
#include <vector>
#include <string>

using namespace std;

int main() {
    vector<string> input;
    string temp;
    while (cin >> temp) {
        input.push_back(temp);
    }
    
    string num = "";
    for (int i = 0; i < input.size(); i++) {
        num += input[i];
    }
    
    if (stoi(num) % 4 == 0) {
        cout << "YES" << endl;
    }
    else {
        cout << "NO" << endl;
    }
    
    return 0;
}